docs/email/base/typography.html

```diff
@@ -150,6 +150,29 @@ <h2 style="font-weight: bold; font-size: 21px; line-height: 24px; color: #242629
         </div>
     </div>
 </section>
+<section class="stacks-section">
+    {% header h2 | Blockquotes %}
+    <div class="stacks-preview mb16">
+{% highlight html linenos %}
+<blockquote style="margin: 0 0 15px; padding: 4px 10px; background-color: #FFF8DC; border-left: 2px solid #ffeb8e;">
```

Comment:
Out of curiosity: for `code` blocks in the previous PR you've defined font `color`. Here, for `blockquote` you're not doing so. Any reason for (not?) doing that?

Reply:
Yes this is intentional. `<code>` can be used with or without a preceding `<pre>` tag, so it may not inherit the color it needs. The `<p>` tag is likely already set up to inherit the color from the rest of the email body, so I've left it out.

But good point! If, for some reason the text around the `<blockquote>` is styled differently, the `<blockquote><p>` may need an inline style. Though I don't see that being an issue in our case. Something to keep an eye on, though.

Reply:
It's good to know, thanks!